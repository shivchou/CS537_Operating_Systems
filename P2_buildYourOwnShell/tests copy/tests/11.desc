Tests for Builtin Change Directory
