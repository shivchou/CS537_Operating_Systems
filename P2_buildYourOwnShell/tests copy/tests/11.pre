mkdir -p /tmp/testcd/subdir; echo "hello" > /tmp/testcd/file.txt
