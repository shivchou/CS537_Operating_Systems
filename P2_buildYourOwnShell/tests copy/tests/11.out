/tmp
/tmp/testcd
/tmp/testcd/subdir
/tmp/testcd/subdir
/root
/tmp
